examples/mqtt/mqtt-shapes/README.md

```diff
@@ -276,7 +296,7 @@ Also, make sure that `mosquitto_pub`, and `mosquitto_sub` are available from
 the current command prompt, otherwise you should modify the PATH environment
 variable.
 
-Linux® systems:
+Linux and macOS systems:
```

Comment:
I think the previous paragraph (from "The following commands..." onward) up to the "NOTE" below (included) should be moved to the next section ("Navigating the demo with tmux").

That way this section can be focused on presenting the manual steps.

Both the moved paragraph and this section might need some small rephrasing to make them flow better after the "surgery".

Reply:
Sounds good